depthcharge/components/encoders.py

```diff
@@ -28,14 +28,14 @@ def __init__(self, dim_model, min_wavelength=0.001, max_wavelength=10000):
             base = min_wavelength / (2 * np.pi)
             scale = max_wavelength / min_wavelength
         else:
-            base = 1
-            scale = max_wavelength / (2 * np.pi)
+            base = 1 / (2 * np.pi)
+            scale = max_wavelength
 
-        sin_term = base * scale ** (
-            torch.arange(0, n_sin).float() / (n_sin - 1)
+        sin_term = base * (
+            scale ** (torch.arange(0, n_sin).float() / (n_sin - 1))
```

Comment:
**question:** Are we missing the multiplication by 2 in the exponent of the `scale` here?

Reply:
Yes - but the 2 actually originates from the denominator being `d/2`. This should be equivalent because n_sin = n_cos = d/2 (at least for even numbers), but I'm going to make the change so that it exactly matches the equation.